editors/MacVim/files/patch-remove-updater.diff

```diff
@@ -14,21 +14,31 @@
                              </menuItem>
 --- src/MacVim/Base.lproj/Preferences.xib	2021-04-05 16:02:16.000000000 +0200
 +++ src/MacVim/Base.lproj/Preferences.xib	2021-04-05 16:04:16.000000000 +0200
-@@ -10,10 +10,8 @@
-             <connections>
+@@ -11,13 +11,11 @@
                  <outlet property="advancedPreferences" destination="620" id="632"/>
+                 <outlet property="allowForceClickLookUpButton" destination="rlt-zw-mfW" id="xCv-HS-zyJ"/>
                  <outlet property="appearancePreferences" destination="hr4-G4-3ZG" id="G54-DD-ACh"/>
 -                <outlet property="autoInstallUpdateButton" destination="UYM-W0-Kgl" id="cX5-tk-9WJ"/>
                  <outlet property="generalPreferences" destination="115" id="143"/>
+                 <outlet property="inputPreferences" destination="Bnq-Nx-GJH" id="FES-rQ-Fpa"/>
                  <outlet property="layoutPopUpButton" destination="427" id="596"/>
+                 <outlet property="sparklePrereleaseButton" destination="4Y5-mA-blQ" id="baN-Dn-dgO"/>
+                 <outlet property="sparklePrereleaseDesc" destination="ere-oJ-WLd" id="fTe-GS-5qR"/>
```

Comment:
These buttons seem related to Sparkle... should we remove them as well?

Reply:
I updated the PR to remove `sparklePrereleaseButton` and `sparklePrereleaseDesc`.